packages/finance/src/price/dto/mod.rs

```diff
@@ -194,6 +194,53 @@ where
         Base: Currency;
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct BasePrice<QuoteC, BaseG>
```

Comment:
please:
- extract it into its own module `finance::price::base`. Rationale to be outside of `dto` is that it is not used for data transfer, no ser-de implemented, it is just another form of `finance::price::Price`
- change the places of the two type parameters to reflect the order in `finance::price::Price` and of their usage in the data members